src/compiler/transformers/classFields.ts

```diff
@@ -2416,11 +2416,11 @@ export function transformClassFields(context: TransformationContext): (x: Source
             factory.createBlock(
                 setTextRange(
                     factory.createNodeArray(statements),
-                    /*location*/ constructor ? constructor.body!.statements : node.members,
+                    /*location*/ constructor?.body ? constructor.body.statements : node.members,

```

Comment:
```suggestion
                    /*location*/ constructor?.body?.statements ?? node.members,
```

This is also valid, right?

Reply:
ye, sure